packages/ui-components/src/components/Chart/BarChart.js

```diff
@@ -39,6 +40,13 @@ export const BarChart = ({
         return 1;
       }
     }
+    /*
+      There is a known bug with Recharts where if a bar graph is a time scale, it overflows the bars over the edge of the axes. 
```

Comment:
Really clear note, thanks @acdunham! Could you please also include a link to the issue in the recharts repo?

Reply:
Added :)